### DISCUSSION TITLE
Issue with Importing 'main' Property in Deno in vscode

### DISCUSSION THREAD

**Discussion by hetari** [denoland/deno#26398] [Q&A] [answered]
I'm facing this error on vscode:
![image](https://github.com/user-attachments/assets/cee89988-51d3-457e-ac44-50c294e1ef01)

But when I run the code with:
```bash
deno run main.ts
```
it seems to work
![image](https://github.com/user-attachments/assets/199349fc-526a-4953-b6d5-636404ea3dfe)

How do we hide this annoying error in vscode?

**Comment by marvinhagemeister** [comment, marked as answer]
Do you have installed the Deno vscode extension?

**Comment by hetari** [comment]
Thanks, I installed it and this fixes it.
